DigitalLearningSolutions.Web/Views/Home/Welcome.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.Home
+@model LandingPageViewModel
+
+@{
+  ViewData["Title"] = "Welcome";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/home/welcome.css")" asp-append-version="true"/>
+
+<div class="nhsuk-grid-row">
+  <div id="column" class="nhsuk-grid-column-full">
+    <h1 id="page-heading">A platform for digital learning delivery</h1>
+
+    <p id="page-subheading">Enabling health and care organisations to deliver digital learning to staff and students.</p>
+
+    @if (Model.UserIsLoggedIn) {
+      <a class="nhsuk-button auth-button auth-button--left" href="@Model.CurrentSiteBaseUrl/Home?action=appselect">
+        Application&nbsp;selector
+      </a>
+    } else {
+      <a class="nhsuk-button auth-button auth-button--left" asp-controller="Login" asp-action="Index">
+        Log in
+      </a>
+      <a class="nhsuk-button nhsuk-button--secondary auth-button" asp-controller="Register" asp-action="Index">
+        Register
+      </a>
+    }
+
+    @await Component.InvokeAsync(
+      typeof(ContentsListViewComponent),
+      new { miniHubNavigationModel = Model.MiniHubNavigationModel })
+
+    <p>Our products and services can help you manage digital learning delivery for the benefit of the people in your organisation.</p>
+
+    @await Component.InvokeAsync(typeof(PaginationViewComponent), new { miniHubNavigationModel = Model.MiniHubNavigationModel })

```

Comment:
In general, I think the tag-helper syntax for vcs is much nicer: https://docs.microsoft.com/en-us/aspnet/core/mvc/views/view-components?view=aspnetcore-3.1

I think we already use it for back button in the LearningContent page. :)